web/src/app/shared/models/classification-summary.ts

```diff
@@ -0,0 +1,23 @@
+export interface ClassificationSummary {
+  classificationId?: string;
+  key?: string;
+  applicationEntryPoint?: string;
+  category?: string;
+  domain?: string;
+  name?: string;
+  parentId?: string;
+  parentKey?: string;
+  priority?: number;
+  serviceLevel?: string;
+  type?: string;
+  custom1?: string;
+  custom2?: string;
+  custom3?: string;
+  custom4?: string;
+  custom5?: string;
+  custom6?: string;
+  custom7?: string;
+  custom8?: string;
+}
+
+export const customFieldCount: number = 8;
```

Comment:
Is this always 8? Is there a way to avoid this magic  number?

Reply:
This variable exists because in the implementation we iterate over the 8 custom fields this class exposes.
Whenever we add more custom fields to the classification summary we have to update this number aswell.

Reply:
Is there maybe a way to dynamically generate this number, or would this be too much/ not worth the effort?

Reply:
I have no idea how we should do this. We can timebox this and try it out (in a different ticket).

Reply:
https://taskana.atlassian.net/browse/TSK-1311